packages/types/src/asset-types.ts

```diff
@@ -128,6 +128,18 @@ export const AssetCategories = [
   "defi",
   "meme",
   "liquid_staking",
+  "built_on_osmosis",
+  "sail_initiative",
+  "bridges",
+  "nft_protocol",
+  "ai",
+  "stablecoin",
+  "privacy",
+  "social",
+  "oracles",
+  "dweb",
+  "rwa",
+  "gaming",
```

Comment:
Once we merge https://github.com/osmosis-labs/osmosis-frontend/pull/3126, we can remove these changes (that PR deletes this const var)